cognite/client/_api/data_modeling/instances.py

```diff
@@ -1091,6 +1091,14 @@ def list(
                 >>> c = CogniteClient()
                 >>> instance_list = c.data_modeling.instances.list(limit=5)
 
+            List instances and sort by some property:
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes.data_modeling import InstanceSort
+                >>> c = CogniteClient()
+                >>> property_sort = InstanceSort(property=myView.as_property_ref("some_property"), direction="descending", nulls_first=True)
+                >>> instance_list = c.data_modeling.instances.list(sort=property_sort)
```

Comment:
@haakonvt , any idea why "Test Core" fails because of "NameError: name \'property_sort\' is not defined\n\n')]"  ?

Reply:
Probably because `myView` is not defined anywhere

Reply:
Sorry for all the GH spam, the docstring test behaves in mysterious ways... This PR is ready now, I beliebe. Any more comments before going from draft to normal PR?